test/onnx/onnx_test_common.py

```diff
@@ -190,6 +224,7 @@ def run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
                 ]
             ]
         ] = None,
+        skip_dynamic_shapes_check: Optional[bool] = False,
```

Comment:
We should add `xfail` decorator to the test if it fails dynamic shape check, and add reason/context w/ fixme instead of using this argument.

But I can see you want to have test passing assurance barring this bug right now. Let's make that clear in docstring then.

This is yet another motivation for us to have raiseRegex style xfail for our testing infra.

Reply:
What is your suggestion for docstring change?